### ISSUE TITLE
Support BlastEm OPN2

### ISSUE THREAD

**Issue by jpcima** [Wohlstand/libOPNMIDI#79]
>BlastEm is an open source, higly accurate emulator for the Genesis/Megadrive that runs on modest hardware. It is the first emulator to properly run Titan's impressive Overdrive 2 demo and it is the only other emulator besides Genesis Plus GX that can properly run the original Overdrive. It is the only emulator other than Exodus that passes Nemesis' VDP FIFO Testing ROM, can display "Direct Color DMA" demos and emulates CRAM contention (aka CRAM dots).

Emulator license is GPLv3.
This is supposed to be high-accuracy, and SSG-EG supported according to this (no additional info). https://www.retrodev.com/blastem/trac/ticket/2

Located here:
https://www.retrodev.com/repos/blastem/file/tip/ym2612.h
https://www.retrodev.com/repos/blastem/file/tip/ym2612.c

**Comment by Wohlstand** [comment]
Oh, nice!
And yeah, to avoid name conflicts, it needs a minor refactoring of internality...
Will check it more detailed once I'll reach my home...

**Comment by jpcima** [comment]
Yep. While researching how I may fix neko project SSG-EG, the report came up in search, and I found this.

Unrelated matter: I have noticed that mucom88 updates have introduced some custom modifications into fmgen, which I have not yet examined in due detail. (this being the same core as np2)
